core/Enclave/enclave_hsm.edl

```diff
@@ -105,6 +105,9 @@ enclave {
                                             [out, size=plaintext_len] uint8_t *plaintext, uint32_t plaintext_len,
                                             [out] uint32_t *req_plaintext_len);
 
+        public sgx_status_t enclave_create_apikey([out, size=apikey_len] uint8_t *p_apikey,
```

Comment:
public sgx_status_t enclave_create_apikey([out, size=32] uint8_t *p_apikey);

Reply:
It will be change